apcd_cms/src/client/src/components/Forms/Registrations/RegistrationForm.tsx

```diff
@@ -184,6 +185,17 @@ const initialTouched = {
   ],
 };
 
+const dateThreshold = () => {
```

Comment:
Can we add this required business logic documentation somewhere in confluence with maybe a comment here so future devs know why specifically Oct 1?